PterodactylExample/PterodactylExampleUITests/PterodactylExampleUITests.swift

```diff
@@ -35,6 +35,32 @@ class PterodactylExampleUITests: XCTestCase {
         XCTAssertEqual(defaultsValue.label, testValue.uuidString)
     }
 
+    func testDeleteUserDefaults() throws {
+        let pterodactyl = Pterodactyl(targetAppBundleId: "com.mattstanford.PterodactylExample")
+        let testValue = UUID()
+        pterodactyl.updateDefaults([
+            "Test": .string(testValue.uuidString)
+        ])
```

Comment:
Before we start the app, we add a userDefaults value for `Test` that is the string value of a UUID.